File: Codechef/Practice/Roller Coaster.cpp
https://www.codechef.com/practice/PCPP07/problems/CPPCC07




#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	std::cin >> t;
	while(t>0){
	    int a,b;
	    std::cin >> a>>b;
	    if(a >=b)
	        std::cout << "YES" << std::endl;
	    else
	        std::cout << "NO" << std::endl;
	    t--;
	}
	return 0;
}

File: Codechef/Practice/Roller Coaster.py
https://www.codechef.com/practice/course/python/PPY10/problems/PYDEBUG07




n = int(input())

for i in range(n):
    X, Y = map(int,input().split())
    if X>=Y:
        print("YES")
    else:
        print("NO")